### ISSUE TITLE
more robust service start

### ISSUE THREAD

**Issue by g2flyer** [hyperledger-labs/private-data-objects#182]
- move pgrep from various test scripts into *-start.sh (including adding it also to ss-start.sh where previously we did not have)
- add a test in es-start.sh when (maybe optionally/based on option) returns only once all services are up and running ...

**Comment by cmickeyb** [comment]
duplicate of #164
